### ISSUE TITLE
Pi 4 doesn't seem to be compatible with Buster on a fresh install

### ISSUE THREAD

**Issue by sapnho** [helgeerbe/picframe#364]
Last night, I updated [one of my earliest articles](https://www.thedigitalpictureframe.com/how-to-set-up-your-raspberry-pi-for-your-digital-picture-frame/) to reflect the fact that Buster isn't available in the Raspberry Pi Imager software anymore. Upon testing, I discovered that the Pi 4 apparently now requires Bullseye or Bookworm.

I also saw [this post](https://forums.raspberrypi.com/viewtopic.php?t=362498&sid=6542a2dd8b9dcd3ce45a41813a84f2d4) on the Pi forum.

Good thing, we almost have Pi3D PictureFrame running on Bookworm.

Can anybody confirm the Pi 4 incompatibility with Buster?

**Comment by sapnho** [comment]
I found a more recent Buster image that works on the Pi4: https://downloads.raspberrypi.org/raspios_arm64/images/raspios_arm64-2021-05-28/

That works!

**Comment by Dropbear67** [comment]
> Can anybody confirm the Pi 4 incompatibility with Buster?

yes I can. There was a way around it by overwriting files in the boot directory but hopefully your new image fixes the need to do that

**Comment by sapnho** [comment]
Yes, it does.

**Comment by sapnho** [comment]
I guess we can close this issue.
